packages/commonwealth/server/migrations/20231212140801-divastaking-community-and-contract-update.js

```diff
@@ -0,0 +1,764 @@
+              name: '',
+              type: 'bool',
+            },
+          ],
+          stateMutability: 'view',
+          type: 'function',
+        },
+        {
+          inputs: [],
+          name: 'version',
+          outputs: [
+            {
+              internalType: 'string',
+              name: '',
+              type: 'string',
+            },
+          ],
+          stateMutability: 'view',
+          type: 'function',
+        },
+        {
+          inputs: [],
+          name: 'votingDelay',
+          outputs: [
+            {
+              internalType: 'uint256',
+              name: '',
+              type: 'uint256',
+            },
+          ],
+          stateMutability: 'view',
+          type: 'function',
+        },
+        {
+          inputs: [],
+          name: 'votingPeriod',
+          outputs: [
+            {
+              internalType: 'uint256',
+              name: '',
+              type: 'uint256',
+            },
+          ],
+          stateMutability: 'view',
+          type: 'function',
+        },
+        {
+          inputs: [],
+          name: 'proposalThreshold',
+          outputs: [
+            {
+              internalType: 'uint256',
+              name: '',
+              type: 'uint256',
+            },
+          ],
+          stateMutability: 'view',
+          type: 'function',
+        },
+      ];
+      const abiData = JSON.stringify(abi);
+
+      const hashInstance = hasher({
+        coerce: true,
+        sort: true,
+        trim: true,
+        alg: 'sha256',
+        enc: 'hex',
+      });
+
+      const abiHash = hashInstance.hash(JSON.parse(abiData));
+      await queryInterface.sequelize.query(
+        `
+        INSERT INTO "ContractAbis" ("abi", "abi_hash", "verified", "nickname", "created_at", "updated_at")
+        VALUES (:abi, :abiHash, :verified, :nickname, NOW(), NOW())
+      `,
+        {
+          replacements: {
+            abi: abiData,
+            abiHash: abiHash,
+            verified: true,
+            nickname: 'GovernorCountingSimple',
+          },
+          type: Sequelize.QueryTypes.INSERT,
+          transaction,
+        },
+      );
+
+      // Retrieve the ID of the newly inserted "ContractAbi" (if needed for further operations)
+      const contractAbiId = await queryInterface.sequelize.query(
+        `
+        SELECT id FROM "ContractAbis" WHERE nickname = 'GovernorCountingSimple'
+      `,
+        {
+          type: Sequelize.QueryTypes.SELECT,
+          transaction,
+        },
+      );
+
+      // Assuming you need the ID for the next operation
+      await queryInterface.sequelize.query(
+        `
+        UPDATE "Contracts"
+        SET "abi_id" = :abiId
+        WHERE id = 710
+      `,
+        {
+          replacements: { abiId: contractAbiId[0].id },
+          type: Sequelize.QueryTypes.UPDATE,
+          transaction,
+        },
+      );
+
+      await transaction.commit();
```

Comment:
NIT: This try catch is not needed.

If the transaction scope finishes, sequelize auto-commits the transaction. If there is an error thrown within the transaction scope, sequelize will auto-roll back the transaction